documentation/samplefiles/readme.go.md

```diff
@@ -12,35 +12,15 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2019-12-01
-  - tag: package-2020-07-01-preview
-  # add every tag listed below
+  - tag: package--[[Version]]-[[ReleaseState]]
```

Comment:
`-[[ReleaseState]]` should be unnecessary. @ArcturusZhang could you help check the sample go readme file? `[[ReleaseState]]` is either `stable` or `preview`

Reply:
extra hyphen here
```suggestion
  - tag: package-[[Version]]-[[ReleaseState]]
```

Reply:
Well @leni-msft I suppose `[[ReleaseState]]` should be necessary to emphasize the state of this tag. But I suppose this should align with the tags defined in `readme.md`

Reply:
@leni-msft  @ArcturusZhang   the placeholders are matches the openapi hub portal's code. You can view the placeholder specific meaning in https://github.com/JackTn/azure-rest-api-specs/blob/dev-tianenattestation-Microsoft.TianenAttestation-2020-09-27/specification/tianenattestation/resource-manager/readme.csharp.md

Reply:
@ArcturusZhang  let me know if you have any questions

Reply:
@JackTn as discussed, the [[ReleaseState]] is redundant, so remove it from tag.